src/guides/v2.3/howdoi/checkout/checkout_payment.md

```diff
@@ -308,5 +308,24 @@ In your custom module directory, create a new `<your_module_dir>/view/frontend/l
     </body>
 </page>
 ```
+## Step 5:Run the CLI Commands {#compile-code-deploy-contents-and-clean-the-cache}
```

Comment:
Hi @VinothKumar361. Thank you for your collaboration. 

I'm not sure we need this information trough the documentation. The DevDocs is a resource with the Magento development-related documentation. The target audience is Magento developers. 

If you are a Magento developer, you are supposed to know the very basic development routine like cache cleaning, running your development environment, using IDE/browser before creating working solutions. So, posting this info over and over again on all pages (it applies for most of the code change operations though) might not be as useful as we want it to see. 

Also, if we are talking about Magento development, the developer's environment is supposed to be in the `development` mode which does not require DI compilation and static content deployment. On production environments, the mentioned operations should be a part of the deployment process.

Reply:
You make a good point, Rogyar. There have been a few PRs adding this same info to other topics. I would rather lean on the side of providing more information than necessary, if it can prevent confusion to users. Good feedback.

Reply:
From the other hand, providing more information than necessary leads to future search and maintenance issues.
If the information is relevant and must be present in several topics, consider using the `{% include %}` functionality to follow the docs-like DRY principle.